build/buttercup-desktop.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Version=1.1
```

Comment:
@sum01 Is this the `.desktop` file version or the app version?

Reply:
That version is optional, but it defines what spec version of freedesktop's desktop spec the file conforms to. It has nothing to do with the project version.